wiki/Reporting_Bad_Behaviour/Handling_Cheating/en.md

```diff
@@ -1,74 +1,72 @@
 
-Stay calm.
+## I found someone who is definitely cheating. I can tell by their replay/score! What should I do?
+Stay calm and **don't discuss their play in public chat**.
+Don't give them undue attention or ask others to spectate them. There are three ways to report users.
 
-**DON'T discuss their play in public chat**.
+### User Profile
+At the user's profile, a `Report User` button will appear when you move your cursor to their avatar. Write any information that can prove that the user is doing misbehaviour.
 
-Don't give them undue attention or ask others to spectate them.
+*Take note of the selected game mode before playing*
 
-Click the "Report" link at the right side of their score on the beatmap info (website)
+### Beatmap Page
+In beatmap pages, you can report a user in the top leaderboards by clicking the `Report` button at right side.
 
-Use !report in in-game chat to send a message to moderators.
+### In-game chat
+Use the `!report` command in any channel to inform the chat moderators. Other players will not see your reports in chat.
 
-`Example: !report `<Username>` is suspected to be a cheater.`
+      Format: !report <username> <reason>
+     Example: !report this_user suspected cheating
 
-Your help in spotting foul play is much appreciated. You can also send an email directly to osu@ppy.sh for a relatively fast response.
+Your help in spotting foul play is much appreciated. You can also send an email directly to [osu@ppy.sh](mailto:osu@ppy.sh) for a relatively fast response.
 
-Is using specialised hardware counted as cheating?
---------------------------------------------------
 
+## Is using specialized hardware counted as cheating?
 Peripherals such as tablets and touchscreens are fine. Anything which decreases the skill required to play is not. This means that using multiple PCs, input splitters or anything of that kind is considered a no-no.
 
 Obviously, this is hard to monitor unless we are standing next to you, but if you have been accused of such practices, making a video (using a camera) of your play on a map may help clear the air.
 
-How can I request a ban appeal?
--------------------------------
 
-Address your emails to accounts@ppy.sh and use a subject of "Ban Appeal: <username>"
+## How can I request a ban appeal?
+Address your emails to [accounts@ppy.sh](mailto:accounts@ppy.sh) and use a subject of `Ban Appeal: <username>`
 
-Include the reason you were banned (if you have been banned, you know why) and explain why you do not believe your behaviour should have resulted in a ban.
+Include the reason you were banned (if you have been banned, you know why) and explain why you do not believe your behavior should have resulted in a ban.
 
-`Allow up to five days for a response. If you make an appeal and it is rejected, all further appeals will be rejected.`
+*Allow up to five days for a response. If you make an appeal and it is rejected, all further appeals will be rejected.*
```

Comment:
Add a section along the lines of:

## What can I do if I've broken the rules?

If you've broken the rules and you'd like to come clean, please contact [accounts@ppy.sh](mailto:accounts@ppy.sh) and explain what you've done. Honesty is very much appreciated and this will likely not result in a permanent punishment.